Source/Core/AudioCommon/OpenALStream.cpp

```diff
@@ -146,18 +146,39 @@ void OpenALStream::SoundLoop()
 	memset(uiBuffers, 0, numBuffers * sizeof(ALuint));
 	uiSource = 0;
 
+	// Checks if a X-Fi is being used. If it is, disable FLOAT32 support.
+	if (strstr(alGetString(AL_RENDERER), "X-Fi") != NULL)
```

Comment:
We prefer `nullptr` over `NULL` (or as another option, I think you can leave out the ` != NULL` part entirely).